mpas_analysis/ocean/climatology_map_bgc.py

```diff
@@ -0,0 +1,356 @@
+# Copyright (c) 2017,  Los Alamos National Security, LLC (LANS)
```

Comment:
@bradyrx, if you don't mind, could you run a PEP8 style checker or alternatively open this file in spyder (and, I think, turn on PEP8 checking) or another IDE that includes PEP8 style checking? We try to stick pretty closely with that standard. Mostly what I'm seeing is lines with trailing whitespace and places without the expected number of line breaks. If you'd rather not deal with this, that's totally fine and I can clean it up later.

Reply:
@xylar fixed whitespace and other PEP8 issues.